release-notes/5.0/netstandard2.1/5.0-preview5_System.Net.WebSockets.md

```diff
@@ -0,0 +1,10 @@
+# System.Net.WebSockets
+
+``` diff
+ namespace System.Net.WebSockets {
+     public abstract class WebSocket : IDisposable {
++        public static bool IsApplicationTargeting45();
```

Comment:
WAT? :smile:

Reply:
sry whts the question here ?. Api is showing here as its absent from netstandard2.1

Reply:
I just found it strange that we have a static API on WebSocket for detecting .NET 4.5.

Reply:
https://github.com/dotnet/standard/issues/111